person.rb

```diff
@@ -1,3 +1,5 @@
+require_relative './nameable'
+
 class Person
   attr_accessor :age, :name, :rental
   attr_reader :id
```

Comment:
- [ ] According to the project requirement please make sure that the Person class inherits from the Nameable class. ie:
`class Person < Nameable`